memory/semantic.py

```
@@ -108,18 +111,23 @@ async def rerank_results(self, query, results: list, k=5) -> list:
 
 class SemanticMemoryOriginal:
     def __init__(self, persist_directory, lanlan_name, name_mapping):
-        config_manager = get_config_manager()
-        api_config = config_manager.get_model_api_config('summary')
-        self.embeddings = OpenAIEmbeddings(base_url=api_config['base_url'], model=SEMANTIC_MODEL, api_key=api_config['api_key'])
-        # self.vectorstore = Chroma(
-        #     collection_name="Origin",
-        #     persist_directory=persist_directory[lanlan_name],
-        #     embedding_function=self.embeddings
-        # )
+        # embeddings 和 vectorstore 当前均未启用（Chroma 已注释），懒初始化以避免启动时无用开销
```

Ruff (0.15.5)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)